### PR TITLE
Defender can now evo into bull.

### PR DESCRIPTION
**Pull request by Mannybrado**
## About The Pull Request
Defender can use darwin-esque evolution to EVOLVE into the Bull Xenomorph caste.

## Why It's Good For The Game
1. More options.
2. The Crusher trilogy is finally complete. It all started with that funny defender charge... 
(edit:) 3. I don't want to play warrior but I want to play defender and have a t2 option that isn't warrior. 
(edit2:) I would add another t2 for sentinel, but unfortunately there are only 4 ranged castes in the game and 2 of them are t3s.

## Changelog
:cl:
tweak: Defender can evo to Bull
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm [hunk 1]
```diff
@@ -27,7 +27,10 @@
 	evolution_threshold = 80
 	upgrade_threshold = TIER_ONE_YOUNG_THRESHOLD
 
-	evolves_to = list(/mob/living/carbon/xenomorph/warrior)
+	evolves_to = list(
+	/mob/living/carbon/xenomorph/warrior,
+	/mob/living/carbon/xenomorph/bull
```
Comment (novaepee) [inline code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm:32]: need comma to not die from liner

/mob/living/carbon/xenomorph/bull,

Comment (Mannybrado) [reply code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm]: guh

**Comment by Carnage068** [comment]
thank u, if this goes through then i won't have to go runner to become bull. if only bull could evolve into rav or something so you don't have to be a squishy caste to go rav, but i'll shut up and be grateful for what we're getting here, lol

**Comment by Mannybrado** [comment]
but why it fail doe

**Comment by KrabSpider** [comment]
Finally. A reason to choose Defender again

**Pull request merged**